composer.json

```diff
@@ -18,15 +18,15 @@
     ],
 
     "require": {
-        "php": "^7.2.5",
+        "php": ">=7.2.5",
         "ext-json": "*",
-        "symfony/http-client": "^4.4|^5.0"
+        "symfony/http-client": "^4.4 || ^5.3 || ^6.0"
     },
 
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.16",
-        "symfony/framework-bundle": "^4.4 | ^5.0",
-        "symfony/phpunit-bridge": "^5.0",
+        "phpstan/phpstan": "^1.2",
+        "phpunit/phpunit": "^8.5 || ^9.5",
```

Comment:
We should use `symfony/phpunit-bridge` - 3rd party libs is where that shines, allowing different php versions more easily. But, we could also worry about that later when we centralize things more.